src/style.css

```diff
@@ -0,0 +1,37 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: fit-content;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
`background-image: url(/images/star-active.svg);` is duplicated, please check the checklist
<img width="1163" alt="Screenshot 2023-09-17 at 16 21 22" src="https://github.com/mate-academy/layout_stars/assets/64746943/7597ae22-9f2c-4f7c-8fb2-5433eccb61d6">